docs/en/latest/admin-api.md

```diff
@@ -276,44 +269,46 @@ After successful execution, status nodes will be updated to:
-| POST   | /apisix/admin/services             | {...}        | Create resource, and ID is generated by server                                                                                                                                                                                                                                                                      |
-| DELETE | /apisix/admin/services/{id}        | NULL         | Remove resource                                                                                                                                                                                                                                                                                                     |
-| PATCH  | /apisix/admin/services/{id}        | {...}        | Standard PATCH. Update some attributes of the existing Service, and other attributes not involved will remain as they are; if you want to delete an attribute, set the value of the attribute Set to null to delete; especially, when the value of the attribute is an array, the attribute will be updated in full |
-| PATCH  | /apisix/admin/services/{id}/{path} | {...}        | SubPath PATCH, specify the attribute of Service to be updated through {path}, update the value of this attribute in full, and other attributes that are not involved will remain as they are. The difference between the two PATCH can refer to the following examples                                              |
+| Method | Request URI                        | Request Body | Description                                                                                                                     |
+| ------ | ---------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| GET    | /apisix/admin/services             | NULL         | Fetches a list of available Services.                                                                                           |
+| GET    | /apisix/admin/services/{id}        | NULL         | Fetches specified Service by id.                                                                                                |
+| PUT    | /apisix/admin/services/{id}        | {...}        | Creates a Service with the specified id.                                                                                            |
+| POST   | /apisix/admin/services             | {...}        | Creates a Service and assigns an id.                                                                                            |
```

Comment:
How about using `random id` and `specified id`?

Reply:
+1